plugins/modules/net_tools/nmcli.py

```diff
@@ -1458,6 +1469,18 @@ def tunnel_conn_type(self):
             'sit',
         )
 
+    @staticmethod
+    def normalise_ip4(ip4_addresses):
+        if ip4_addresses is None:
+            return None
+        return [address if '/' in address else address + '/32' for address in ip4_addresses]
+
+    @staticmethod
+    def normalise_ip6(ip6_addresses):
```

Comment:
Same as above

Reply:
IPv6 normalization is a lot more complicated, since you can use `::`. Also you need to use this function for both what `nmcli` returns and for the user input, to really make sure you end up with the same value.